lines/client/rundown.py

```diff
@@ -1,57 +1,89 @@
 """Client for TheRundown API."""
 import datetime
+import logging
 import os
 import requests
 
+from prefect import Task
+
 from lines.client.config import RundownSportId
-from lines.libs import LinesResponseFacade
+from lines.libs.errors import InvalidSportIdError
+from lines.libs.facades import LinesResponseFacade
+
+logging.basicConfig(level="INFO")
+_LOGGER = logging.getLogger(__name__)
+
+
+class Rundown(Task):
+    """
+    TheRundown API Wrapper Task.
 
+    Rundown is a purely designed API wrapper for TheRundown.io REST API
+    The task will be mapped to fetch lines for many sports
 
-class RundownClient(object):
-    """TheRundown API Wrapper Client."""
+    This will be carried out by interval schedule and mapped according.
 
-    def __init__(self):
-        """Constructor for RundownClient."""
-        self.base_url = "https://therundown-therundown-v1.p.rapidapi.com/"
-        self._api_host = os.getenv("RUNDOWN_API_HOST")
-        self._api_key = os.getenv("RUNDOWN_API_KEY")
-        self._today = datetime.datetime.now()
+    Args:
+        sport: (int) - sport_id value of `RundownSportId`
+    Returns:
+        lines: (LinesResponseFacade) - facade of response object `.json()`
+    """
 
-    def __repr__(self):
-        """Representation for RundownClient."""
-        return "<RundownClient: {}>".format(str(self._today))
+    def __init__(self, *args, **kwargs):
+        """Constructor for Rundown"""
```

Comment:
```
Attributes
----------
xyz (str): Does such and such.
```

Not sure if `----` is used in Google docstrings.  I've been liking `numpy` recently.